src/Nethermind/Nethermind.Evm/TransactionSubstate.cs

```diff
@@ -47,24 +47,32 @@ public TransactionSubstate(EvmExceptionType exceptionType, bool isTracerConnecte
                 {
                     if (Output.Length > 0)
                     {
-                        try
-                        {
-                            BigInteger start = Output.Span.Slice(4, 32).ToUnsignedBigInteger();
```

Comment:
Previously this the exceptions as it went in this path without even checking Output was large enough to slice.

Also produced it when not Tracing but not using NullTxTracer